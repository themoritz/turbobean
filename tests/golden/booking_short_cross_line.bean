2020-01-01 open Assets:Stocks SHORT "FIFO"
2020-01-01 open Assets:Cash

2020-01-02 * "Open short"
  Assets:Stocks  -1 SHORT @ 10 USD
  Assets:Cash    10 USD

2020-01-03 txn "Close short, cross line"
  Assets:Stocks   2 SHORT @ 20 USD
  Assets:Cash
