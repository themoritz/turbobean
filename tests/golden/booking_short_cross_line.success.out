Assets     -30 USD
           1 SHORT @ 20 USD {2020-01-03}
  Stocks   1 SHORT @ 20 USD {2020-01-03}
  Cash     -30 USD
